libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryInitializerTests.java

```diff
@@ -62,18 +74,25 @@ public void telemetryInitializerMiddlewareLogActivitiesEnabled() {
 		.startTest().join();
 
 		// Verify
-		verify(mockTelemetryClient, times(6));
+        verify(mockTelemetryClient, times(6)).trackEvent(
+            eventNameCaptor.capture(),
+            propertiesCaptor.capture()
+            );
```

Comment:
Not sure about this, because C# check the times that `mockTelemetryClient` is called, not matter if it's called using `trackEvent`.

Here, you are checking the `trackEvent`, this is not ok